bionty/models.py

```diff
@@ -182,7 +182,7 @@ def __init__(self, *args, **kwargs):
         if (
             args
             and len(args) == 1
-            and isinstance(args[0], (tuple, list))
+            and isinstance(args[0], tuple | list)
```

Comment:
I don't think this should be changed

Reply:
Why? This does not change the behavior. It's a new feature of Python 3.10.

Reply:
OK, fine, but it feels unnatural to me here, I still prefer the () syntax.